docs/docs/troubleshooting.md

```diff
@@ -55,3 +55,8 @@ keytool -import -trustcacerts -file ./cacert.cer -alias yourca --keystore /my/pa
 ## Error reading authentication secret from file
 
 When using framework-authentication on the master, be sure to set a secret that has a minimum of eight characters. Secrets shorter than that length may not be accepted by Marathon.
+
+## Using firewalls
```

Comment:
Reading the documentation above this point sounds like we're troubleshooting a problem with apps and firewalls. Maybe some restructuring is in order? E.g. `App Troubles` and `Marathon Troubles`? And then add a sentence about what's going on here e.g. `Marathon uses mesos v0 API through a native scheduler driver. This itself uses a libmesos library that opens a connection to mesos master. By default, any port from the ephemeral range is used by libmesos but that can be changed in the presence of a firewall by setting LIBPROCESS_PORT environment variable....`